modules/ols-providing-custom-knowledge-to-the-llm.adoc

```diff
@@ -0,0 +1,111 @@
+// Module included in the following assemblies:
+// * lightspeed-docs-main/configure/ols-configuring-openshift-lightspeed.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="providing-custom-knowledge-to-the-llm_{context}"]
+= Providing custom knowledge to the LLM
+
+Customize the information available to the large language model (LLM) by providing access to a container image that resides in a remote image registry. The examples in this procedure use `quay.io` as the remote container image registry, and the path for the custom image is `quay.io/<username>/my-byok-image:latest`.
+
+:FeatureName: The BYO Knowledge feature
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+
+* You are logged in to the {ocp-product-title} web console as a user with the `cluster-admin` role. Alternatively, you are logged in to a user account that has permission to create a cluster-scoped CR file.
+
+* You have installed the {ols-long} Operator.
+
+* You have configured the large language model (LLM) provider.
+
+* The custom information you want to add resides as a collection of Markdown files; no other file format is supported.
+
+* You have an account for a container image registry, such as `quay.io`.
+
+.Procedure 
+
+. Specify the location of the directory with the markdown files for the custom RAG database and the path for the image that the BYO Knowledge tool generates by running the following command: 
++
+[source,terminal]
+----
+$ podman run -e OUT_IMAGE_TAG=my-byok-image -it --rm --device=/dev/fuse -v <directory_for_markdown_files>:/markdown:Z -v <directory_for_image_tar>:/output:Z $MY_BYOK_TOOL_IMAGE
```

Comment:
Consider splitting the arguments across lines for scannability i.e.
```suggestion
$ podman run \
  -e OUT_IMAGE_TAG=my-byok-image \
  -it --rm --device=/dev/fuse \
  -v <directory_for_markdown_files>:/markdown:Z \
  -v <directory_for_image_tar>:/output:Z $MY_BYOK_TOOL_IMAGE
```
In doing this, I notice it's got two `-v`s - is that right?

Reply:
Yes, ran it by the dev to double-check, too.

done.